src/components/pressables/IconButton.tsx

```diff
@@ -26,7 +26,7 @@ export const IconButton = (props:IconButtonProps): JSX.Element => {
 
 
   return (
-    <Pressable onPress={() => onPress} className={cn(buttonStyles, buttonBackground)} disabled={disabled}>
+    <Pressable onPress={(event) => onPress ? onPress(event) : null} className={cn(buttonStyles, buttonBackground)} disabled={disabled}>
```

Comment:
You can do this like:

`onPress={onPress}`

If it's `undefined` it wont be used and any params get passed through automatically